### PR TITLE
[BUG] Airtable fix computed fields should not appear as editable

### PR DESCRIPTION
**Pull request by jcortes**
### <samp>🤖 Generated by Copilot at 5e4451d</samp>

Updated the Airtable component to handle computed fields and fix bugs. Added a utility function to exclude computed fields from the action props and a constant for the lookup field type. Bumped the versions of the actions and the `@pipedream/airtable` package.

### CODE REVIEW

File: components/airtable/common/utils.mjs [hunk 1]
```diff
@@ -80,6 +80,26 @@ function fieldToProp(field) {
   };
 }
 
+function isComputedField(field) {
```
Comment (dylburger) [inline components/airtable/common/utils.mjs:83]:
```suggestion
function isComputedField(field) {
```

Comment (dylburger) [reply components/airtable/common/utils.mjs:83]: Changed comment to start with a verb

**Review by GTFalcao — APPROVED** [review]
Hi @jcortes , thanks for the fix, looks good to me!

**Comment by vunguyenhung** [comment]
Hi, I can confirm that the computed fields are not shown anymore
image
image

**Pull request merged**